nixos/modules/security/hidepid.nix

```diff
@@ -0,0 +1,32 @@
+{ config, lib, ... }:
+with lib;
+
+{
+  options = {
+    security.hideProcessInformation = {
+      enable = mkEnableOption = ''
+        restrict access to process information to the process owner.  Enabling
+        this option implies, among other things, that users can can only see
+        their own processes and that information such as command-line arguments
+        is not leaked between users.
+
```

Comment:
This needs `<para>...</para>`, otherwise you won't get a paragraph break in the manual.